sql/src/main/java/org/apache/druid/sql/calcite/external/ExternalDataSource.java

```diff
@@ -118,9 +117,10 @@ public boolean isGlobal()
   }
 
   @Override
-  public boolean isConcrete()
+  public boolean isProcessable()
   {
-    return false;
+    // FIXME return false seemed illogical
```

Comment:
agree that `true` makes more sense, since the `external` datasource refers directly to physical files.